### PR TITLE
RCM-118: Fix NPE scenario i noticed on the qa-refapp server

### PR DESCRIPTION
**Pull request by mherman22**
Ticketted at https://openmrs.atlassian.net/browse/RCM-118

### CODE REVIEW

File: omod/src/main/java/org/openmrs/web/controller/report/CohortReportFormController.java [hunk 1]
```diff
@@ -391,11 +391,13 @@ protected ModelAndView onSubmit(HttpServletRequest request, HttpServletResponse
 	}
 	
 	private Node findChild(Node parent, String name) {
-		NodeList list = parent.getChildNodes();
```
Comment (dkayiwa) [inline omod/src/main/java/org/openmrs/web/controller/report/CohortReportFormController.java:394]: To reduce the level of nesting, why not just do the `if parent == null then return early.`

Comment (mherman22) [reply omod/src/main/java/org/openmrs/web/controller/report/CohortReportFormController.java:394]: like i had done in the first commit?

Comment (dkayiwa) [reply omod/src/main/java/org/openmrs/web/controller/report/CohortReportFormController.java:394]: Tru dat!

Comment (mherman22) [reply omod/src/main/java/org/openmrs/web/controller/report/CohortReportFormController.java:394]: done!

**Pull request merged**